File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/PlusGameTournamentManageForm.java
package com.sdkwork.backend.model;

public class PlusGameTournamentManageForm {
    private String status;
    private String signUpStartAt;
    private String signUpEndAt;
    private String startAt;
    private String endAt;
    private Integer winnerUserId;
    private Integer winnerClubId;

    public String getStatus() {
        return this.status;
    }
    
    public void setStatus(String status) {
        this.status = status;
    }

    public String getSignUpStartAt() {
        return this.signUpStartAt;
    }
    
    public void setSignUpStartAt(String signUpStartAt) {
        this.signUpStartAt = signUpStartAt;
    }

    public String getSignUpEndAt() {
        return this.signUpEndAt;
    }
    
    public void setSignUpEndAt(String signUpEndAt) {
        this.signUpEndAt = signUpEndAt;
    }

    public String getStartAt() {
        return this.startAt;
    }
    
    public void setStartAt(String startAt) {
        this.startAt = startAt;
    }

    public String getEndAt() {
        return this.endAt;
    }
    
    public void setEndAt(String endAt) {
        this.endAt = endAt;
    }

    public Integer getWinnerUserId() {
        return this.winnerUserId;
    }
    
    public void setWinnerUserId(Integer winnerUserId) {
        this.winnerUserId = winnerUserId;
    }

    public Integer getWinnerClubId() {
        return this.winnerClubId;
    }
    
    public void setWinnerClubId(Integer winnerClubId) {
        this.winnerClubId = winnerClubId;
    }
}

File: sdkwork-backend-sdk-typescript/src/types/plus-game-tournament-manage-form.ts
/** æ£çæ¸¸æèµäºç®¡çè¡¨å */
export interface PlusGameTournamentManageForm {
  /** èµäºç¶æ */
  status?: 'DRAFT' | 'REGISTERING' | 'RUNNING' | 'SETTLING' | 'FINISHED' | 'CANCELED';
  /** æ¥åå¼å§æ¶é´ */
  signUpStartAt?: string;
  /** æ¥åç»ææ¶é´ */
  signUpEndAt?: string;
  /** èµäºå¼å§æ¶é´ */
  startAt?: string;
  /** èµäºç»ææ¶é´ */
  endAt?: string;
  /** å åç¨æ·ID */
  winnerUserId?: number;
  /** å åä¿±ä¹é¨ID */
  winnerClubId?: number;
}
